packages/api/src/external/fhir/shared/json-validator.ts

```diff
@@ -0,0 +1,50 @@
+import Ajv, { ErrorObject } from "ajv";
+import schema from "./fhir.schema.json";
+import metaSchema from "ajv/lib/refs/json-schema-draft-06.json";
+import { Bundle } from "../../../routes/medical/schemas/fhir";
+import BadRequestError from "../../../errors/bad-request";
+
+type Error = {
+  resourceType: string;
+  errors: ErrorObject[] | null | undefined;
+};
+
+export const validateFhirEntries = (bundle: Bundle): Bundle => {
+  const ajv = new Ajv({
+    // I need to use strict as an option but its not in the AJV.Options
+    // eslint-disable-next-line
+    // @ts-ignore
+    strict: false,
+  });
+
+  ajv.addMetaSchema(metaSchema);
+
+  const errors: Error[] = [];
+
+  for (const resource of bundle.entry) {
+    const resourceType = resource.resource.resourceType;
+
+    const validate = ajv.compile(schema);
```

Comment:
Can't we compile it outside of this function so we don't need to do this for all entries and calls to this function?